src/platform/loader-base.ts

```diff
@@ -8,7 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 import {assert} from './assert-web.js';
-import {fetch} from './fetch-web.js';
+import {fetch} from './fetch-node.js';
```

Comment:
Wait, won't this break the custom esm-loader script which looks for -web and converts to -node? How will this work on browsers now?

Reply:
Note the "DO NOT MERGE". This is not intended to be a PR, it's just me sharing code with Sarah.

I would like to fix the actual problem though, which is that the web packed language server no longer runs properly (though it used to).

It seems that there's been a change somewhere outside of the language server that is breaking it.